membership/src/main/scala/zio/membership/Main.scala

```diff
@@ -0,0 +1,44 @@
+package zio.membership
+
+import zio._
+import zio.duration._
+import zio.macros.delegate._
+import zio.membership.transport.tcp
+import zio.membership.hyparview.HyParView
+import zio.nio.SocketAddress
+import upickle.default._
+import zio.nio.InetSocketAddress
+
+object Main extends zio.App {
+
+  override def run(args: List[String]) = {
+    val env =
+      ZManaged.environment[ZEnv] @@
+        tcp.withTcpTransport(64, 100.seconds, 100.seconds)
+    implicit val rw: ReadWriter[InetSocketAddress] =
+      macroRW[(String, Int)]
+        .bimap[InetSocketAddress](
+          i => (i.hostString, i.port),
+          { case (host: String, port: Int) => unsafeRun(SocketAddress.inetSocketAddress(host, port)) }
+        )
+
+    SocketAddress
+      .inetSocketAddress(8000)
+      .flatMap { addr =>
+        (env >>> HyParView(
+          addr,
+          10,
```

Comment:
Let's try to reduce magic values.

Reply:
This is just an example main in order to play with cluster behavior. I'll remove them completely soonish